src/components/lastRestaurants/lastRestaurants.css

```diff
@@ -0,0 +1,61 @@
+.last-restaurants {
+  padding-top: 20px;
```

Comment:
In this case, I think you can use margin  instead of padding, because isn’t an intern spacing